src/os/exec_posix.go

```diff
@@ -33,3 +34,55 @@ func (p *Process) release() error {
 	runtime.SetFinalizer(p, nil)
 	return nil
 }
+
+// Combination of fork and exec, careful to be thread safe.
+
+// https://cs.opensource.google/go/go/+/master:src/syscall/exec_unix.go;l=143?q=forkExec&ss=go%2Fgo
```

Comment:
can you comment this in a way that describes the compromises you made instead of simply calling it "hacky"?

why is it hacky?